### PR TITLE
[link-validator] Missing file context-api.md

### PR DESCRIPTION
**Pull request by simoneonofri**
Hello,

There is a note with a broken reference link.

`Note: This command lets automation verify that the
   [context api](https://github.com/fedidcg/FedCM/blob/main/proposals/context-api.md)
    was applied properly`

I searched inside the repo and could not find it.

I am proposing to remove it, if it not exist anymore. Otherwise feel free to modify the PR.

Thanks,

Simone

P.S. This should be the last check for publication in `/TR`

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -2412,10 +2412,6 @@ The [=remote end steps=] are:
     </table>
 </figure>
 
-Note: This command lets automation verify that the
```
Comment (npm1) [inline spec/index.bs:2415]: The correct solution is to point to https://w3c-fedid.github.io/FedCM/#dom-identitycredentialrequestoptions-context, not remove the note.

Comment (simoneonofri) [reply spec/index.bs:2415]: ok i edited the code to have the correct link and updated also the page on TR

Comment (npm1) [reply spec/index.bs:2415]: It looks like PR preview is no longer working on the repo. I think this link works but there is no preview to double check

File: spec/index.bs [hunk 2]
```diff
@@ -2412,9 +2412,9 @@ The [=remote end steps=] are:
...
-    [context api](https://github.com/fedidcg/FedCM/blob/main/proposals/context-api.md)
-    was applied properly
+ Note: This command lets automation verify that the
+  [context api](#dom-identitycredentialrequestoptions-context)
+     was applied properly
```
Comment (npm1) [inline spec/index.bs:2417]: why the extra space here?

**Review by npm1 — APPROVED** [review]

File: spec/index.bs (hunk 2 above)
Comment (simoneonofri) [reply spec/index.bs:2417]: @npm1 none specifically, removed

**Pull request merged**